docs/concepts/penalty.rst

```diff
@@ -0,0 +1,78 @@
+.. _penalty_sdk:
+
+==============
+Penalty Models
+==============
+
+Penalty models represent constraints as small models such as :ref:`bqm_sdk`\ s
+that have higher values for infeasible states (values of variables that violate
```

Comment:
`values`? Seems like a slightly strange word in this context. I suppose you're trying to avoid "energies"?

Reply:
Yes, for the more abstract models---outside the context of a particular physical system such as a BQM embedded on a QPU where energies are straightforward---users might be more familiar with a generic cost function interpretation.

Reply:
I would probably use the word "cost" then. "value" seems overloaded to me but :shrug: